protocol-definitions/ReplicatedMap.yaml

```diff
@@ -820,3 +820,113 @@ methods:
             since: 2.0
             doc: |
               Number of entries affected by this event.
+  - id: 19
+    name: putAllWithMetadata
+    since: 2.7
+    doc: |
+      Copies all of the mappings from the specified entry list to this map, including metadata.
+      This method uses ReplicatedRecordStore.putRecords method with version=current version + 1.
```

Comment:
What do you mean by `current version`? How does it work when it is concurrent invocations?

Reply:
putRecords is used to migrate all records from one member to another member, every store has some current version that is increased for every modification (I believe). That's why we need the version information here

Reply:
Btw the description is wrong here, we should be just putting records with the provided version. Let me fix here

Reply:
Probably left stale because I based on another protocol PR in this PR